### PR TITLE
Use junit Category annotations to specifiy test execution

### PR DESCRIPTION
**Pull request by jakubschwan**

### CODE REVIEW

File: test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/smoke/ExternalMavenRepoIntegrationTest.java [hunk 1]
```diff
@@ -118,30 +141,6 @@ public void testProcessFromExternalMavenRepo() {
...
-        try {
-            startAndCompleteUsertaskProcess();
-        } catch (KieServicesException ex) {
-            // OK for drools test.
-            assertThat(ex.getMessage()).contains("Server that this client is connected to has no capabilities to handle ProcessServicesClient");
-        }
-
     }
 
-    private void startAndCompleteUsertaskProcess() {
```
Comment (jiripetrlik) [inline test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/smoke/ExternalMavenRepoIntegrationTest.java]: How is this change related to JUnit categories?

Comment (jakubschwan) [reply test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/smoke/ExternalMavenRepoIntegrationTest.java]: Request to split testProcessFromExternalMavenRepo() into two (jBPM and Drools) from PR114.

File: test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/smoke/MultipleProcessesIntegrationTest.java [hunk 2]
```diff
@@ -20,6 +22,7 @@
 import org.kie.server.client.UserTaskServicesClient;
 import org.kie.server.controller.client.KieServerControllerClient;
 
+@Category(Smoke.class)
```
Comment (sutaakar) [inline test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/smoke/MultipleProcessesIntegrationTest.java]: This can be marked as JBPMonly too.

Comment (jakubschwan) [reply test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/smoke/MultipleProcessesIntegrationTest.java]: Yes, it can. But this test run scenario with database and there isn't configuration for drools and database.
So do we want to skip this scenario for drools (skip test in test results) OR not include in drools test run (test is not in test results)?

Comment (sutaakar) [reply test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/smoke/MultipleProcessesIntegrationTest.java]: Good point, I overlooked that. It is ok in current state.
